tests/core/middleware/test_eth_tester_middleware.py

```diff
@@ -20,3 +27,157 @@ def test_get_block_formatters(w3):
     keys_diff = all_block_keys.difference(latest_block_keys)
     assert len(keys_diff) == 1
     assert keys_diff.pop() == "mixHash"  # mixHash is not implemented in eth-tester
+
+
+sample_address_1 = "0x0000000000000000000000000000000000000001"
```

Comment:
nit: Let's have these as capitalized and near the top of the file just to stick with python convention, especially since sync and async both use these values.

```py
from some_module import last_import_statement

SAMPLE_ADDRESS_1 = "0x0000000000000000000000000000000000000001"
SAMPLE_ADDRESS_2 = "0x0000000000000000000000000000000000000002"
```

Reply:
check